src/utils/coinFormatter.js

```diff
@@ -16,26 +16,26 @@ export const dpUI = (amount, dp = VALUE_DECIMALS) => {
 }
 
 export const prettyBalance = (amount, coin, dp = VALUE_DECIMALS) => {
-  if (!amount) return amount
+  if (!amount || !coin) return amount
 
   amount = cryptoassets[coin].unitToCurrency(amount)
 
   return dpUI(amount, dp)
 }
 
 export const prettyFiatBalance = (amount, rate) => {
-  if (!amount) return amount
+  if (!amount || !rate) return amount
```

Comment:
If the rate is not supplied then the calling of this function is meaningless? i'm not sure what it means to return amount. the consumer should not use any values returned

Reply:
Hi, it was because if you send a null/undefined rate we got an error 
`return BN(amount).dividedBy(**rate**).dp(VALUE_DECIMALS, BN.ROUND_FLOOR)`